From 52fec029544562294137641ca6cb48df710536f8 Mon Sep 17 00:00:00 2001
From: Florent Pirou <florent.pirou@intel.com>
Date: Fri, 16 Jan 2026 03:20:05 -0700
Subject: [PATCH 74/78] ipu7 dkms : v4l2-core makefile adaptation 6.18

Signed-off-by: Florent Pirou <florent.pirou@intel.com>
---
 6.18.0/drivers/media/v4l2-core/Makefile | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/6.18.0/drivers/media/v4l2-core/Makefile b/6.18.0/drivers/media/v4l2-core/Makefile
index 2177b9d..82c235f 100644
--- a/6.18.0/drivers/media/v4l2-core/Makefile
+++ b/6.18.0/drivers/media/v4l2-core/Makefile
@@ -2,6 +2,9 @@
 #
 # Makefile for the V4L2 core
 #
+# ipu7-dkms appendix
+CC := ${CC} -I ${M}/../../../include-overrides -I ${M}/../tuners -I ${M}/../dvb-core -I ${M}/../dvb-frontends
+ccflags-y := -DDRIVER_VERSION_SUFFIX=\"${DRIVER_VERSION_SUFFIX}\"
 
 ccflags-y += -I$(srctree)/drivers/media/dvb-frontends
 ccflags-y += -I$(srctree)/drivers/media/tuners
-- 
2.43.0

