From 7f1a6a024fc7c66a152b45ebd1215cc16029b6b4 Mon Sep 17 00:00:00 2001
From: Florent Pirou <florent.pirou@intel.com>
Date: Fri, 16 Jan 2026 03:22:26 -0700
Subject: [PATCH 5/9] drivers: media: set v4l2_subdev_enable_streams_api

Signed-off-by: Florent Pirou <florent.pirou@intel.com>
---
 6.18.0/drivers/media/v4l2-core/v4l2-subdev.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/6.18.0/drivers/media/v4l2-core/v4l2-subdev.c b/6.18.0/drivers/media/v4l2-core/v4l2-subdev.c
index 4f64ac1..b2b7125 100644
--- a/6.18.0/drivers/media/v4l2-core/v4l2-subdev.c
+++ b/6.18.0/drivers/media/v4l2-core/v4l2-subdev.c
@@ -32,7 +32,7 @@
  * 'v4l2_subdev_enable_streams_api' to 1 below.
  */
 
-static bool v4l2_subdev_enable_streams_api;
+static bool v4l2_subdev_enable_streams_api = true;
 #endif
 
 /*
-- 
2.43.0

