From 3ac92b7c62a9c45260e1aee19c14232fad92c7b3 Mon Sep 17 00:00:00 2001
From: florent pirou <florent.pirou@intel.com>
Date: Tue, 28 Apr 2026 07:55:49 -0700
Subject: [PATCH 6/7] ipu-bridge : add sensors

Signed-off-by: florent pirou <florent.pirou@intel.com>
---
 6.17.0/drivers/media/pci/intel/ipu-bridge.c | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/6.17.0/drivers/media/pci/intel/ipu-bridge.c b/6.17.0/drivers/media/pci/intel/ipu-bridge.c
index 3c947d1..e9c8422 100644
--- a/6.17.0/drivers/media/pci/intel/ipu-bridge.c
+++ b/6.17.0/drivers/media/pci/intel/ipu-bridge.c
@@ -51,6 +51,12 @@
  * Please keep the list sorted by ACPI HID.
  */
 static const struct ipu_sensor_config ipu_supported_sensors[] = {
+       /* D3 Embedded ISX031 */
+       IPU_SENSOR_CONFIG("INTC113C", 1, 300000000),
+       /* LI AR0830 */
+       IPU_SENSOR_CONFIG("LIAR0830", 1, 600000000),
+       /* Innodisk AR0822 */
+       IPU_SENSOR_CONFIG("EV8MOOM1", 1, 600000000),
 	/* Himax HM11B1 */
 	IPU_SENSOR_CONFIG("HIMX11B1", 1, 384000000),
 	/* Himax HM2170 */
-- 
2.43.0

